gitbot/lib.py

```diff
@@ -169,8 +169,16 @@ def bump_version(
         run(f"git config user.name {COMMITTER_NAME}", cwd=repo_root)
         run(f"git config user.email {COMMITTER_EMAIL}", cwd=repo_root)
 
-        command = bump_command(ref_sha, author)
-        run(command, cwd=repo_root)
+        try:
+            command = bump_command(ref_sha, author)
+            run(command, cwd=repo_root)
+        except CommandError as e:
+            execution = run("git show", cwd=repo_root)
+            # e.g. https://github.com/getsentry/getsentry/pull/7672/commits
+            if execution.stdout.find(f"getsentry/sentry@{ref_sha}") > -1:
```

Comment:
seems fine -- alternatively we could also make the command in `getsentry` [idempotent](https://youtu.be/o9THkT5ZPi4) by first checking `git diff --quiet` before attempting to commit nothing